scipy/stats/morestats.py

```diff
@@ -1929,8 +1929,8 @@ def levene(*args, **kwds):
     Yci = zeros(k, 'd')
 
     if center not in ['mean', 'median', 'trimmed']:
-        raise ValueError("Keyword argument <center> must be 'mean', 'median'"
-                         + "or 'trimmed'.")
+        raise ValueError("Keyword argument <center> must be 'mean', 'median'" +
```

Comment:
the ``+`` isn't needed, and there should be a space before the last ``"`` otherwise it'll read ``'median'or``